userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -29,6 +29,10 @@ Billing in Advance:: A type of https://docs.killbill.io/latest/catalog-examples.
 
 Billing in Arrear:: A type of https://docs.killbill.io/latest/catalog-examples.html#_billing_modes[billing mode] in which you can charge for services that are delivered in a previous billing period.
 
+[[billing_start_date]]
+Billing Start Date (Or "Billing Date"):: The date on which billing starts for a subscription. See also <<entitlement_start_date, Entitlement Date>>.
```

Comment:
To me, "billing date" sounds like the day on which you are billed. A search of killbill.io for "billing date" shows that we have used it with that meaning, so I would remove "Or 'Billing Date'".